apps/bfd-pipeline/bfd-pipeline-rda-bridge/src/test/java/gov/cms/bfd/pipeline/bridge/model/ModelUtilTest.java

```diff
@@ -0,0 +1,40 @@
+package gov.cms.bfd.pipeline.bridge.model;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import gov.cms.bfd.model.rif.InpatientClaimColumn;
+import java.util.List;
+import org.junit.jupiter.api.Test;
+
+public class ModelUtilTest {
+  /**
+   * Extracts the names of a numbered enum and verifies that the names all fit the same base name
+   * and occur in the list in the correct numbered order.
+   */
+  @Test
+  public void testThatNumberedSequenceNamesAreExtractedInOrder() {
+    final List<String> names =
+        ModelUtil.listNumberedEnumNames(
+            InpatientClaimColumn.values(), InpatientClaimColumn.ICD_DGNS_CD1);
+    assertNamesFollowSequence(names, InpatientClaimColumn.ICD_DGNS_CD1);
+  }
+
+  /**
+   * Lists of names produced by {@link ModelUtil#listNumberedEnumNames} are expected to all have the
+   * exact same name followed by an integer from 1 to the number of items in this list. This helper
+   * method verifies that this pattern holds for the provided list of names and first enum name.
+   *
+   * @param names names produced by {@link ModelUtil#listNumberedEnumNames}
+   * @param firstNameInSequence first enum value whose name appears in the list
+   */
```

Comment:
Thanks for the great documentation here and on the other methods in this PR! Makes it very east to follow what's going on.